src/monster.cpp

```diff
@@ -597,26 +597,67 @@ int monster::print_info( const catacurses::window &w, int vStart, int vLines, in
 {
     const int vEnd = vStart + vLines;
 
-    mvwprintz( w, point( column, vStart ), c_white, "%s ", name() );
+    // display name
+    mvwprintz( w, point( column, vStart ), c_light_gray, _( "Entity : " ) );
+    mvwprintz( w, point( column + 9, vStart ), c_white, name() );
+
+    // display health
+    nc_color color = c_white;
+    std::string sText;
+    get_HP_Bar( color, sText );
+    mvwprintz( w, point( column, ++vStart ), c_light_gray, _( "Health : " ) );
+    mvwprintz( w, point( column + 9, vStart ), color, sText );
+
+    // display sense
+    std::string senses_str = "--";
+    if( sees( g->u ) ) {
+        senses_str = _( "It is aware of your presence" );
+    } else {
+        senses_str = _( "It hasn't noticed you" );
+    }
 
+    mvwprintz( w, point( column, ++vStart ), c_light_gray, _( "Senses : " ) );
+    mvwprintz( w, point( column + 9, vStart ), sees( g->u ) ? c_red : c_green, senses_str );
+
+    // display stance
     const auto att = get_attitude();
-    wprintz( w, att.second, att.first );
+    mvwprintz( w, point( column, ++vStart ), c_light_gray, _( "Stance : " ) );
+    mvwprintz( w, point( column + 9, vStart ), att.second, att.first );
 
-    if( debug_mode ) {
-        wprintz( w, c_light_gray, _( " Difficulty " ) + to_string( type->difficulty ) );
+    // display threat
+    int threatlvl = type->difficulty;
+    nc_color threatlvl_color = c_white;
+    if( threatlvl >= 8 ) {
+        threatlvl_color = c_red;
+    } else if( threatlvl >= 4 ) {
+        threatlvl_color = c_yellow;
+    } else if( threatlvl >= 0 ) {
+        threatlvl_color = c_blue;
     }
 
-    if( sees( g->u ) ) {
-        mvwprintz( w, point( column, ++vStart ), c_yellow, _( "Aware of your presence!" ) );
+    mvwprintz( w, point( column, ++vStart ), c_light_gray, _( "Threat : " ) );
+    mvwprintz( w, point( column + 9, vStart ), threatlvl_color, to_string( threatlvl ) );
+
+    // dipslay aspect
+    mvwprintz( w, point( column, ++vStart ), c_light_gray, _( "Aspect :" ) );
+
+    std::vector<std::string> line1 = foldstring( type->get_description(), getmaxx( w ) - 11 - column );
+    std::vector<std::string> lines = foldstring( type->get_description(), getmaxx( w ) - 1 - column );
+    int numlines = lines.size();
+    for( int i = 0; i < numlines && vStart <= vEnd; i++ ) {
+        if( i == 0 ) {
+            mvwprintz( w, point( column + 9, vStart ), c_white, line1[i] );
+        } else {
+            mvwprintz( w, point( column, ++vStart ), c_white, lines[i] );
+        }
+
```

Comment:
The printed string can be missing characters if more characters are folded into the first line of `lines` than `line1`.

Reply:
thank you, i will look into this

Reply:
i could put it all (the monster description) on a new line, but i kind of prefered like i tried to do.
i will continue trying. but, meanwhile, if you had other suggestions please do tell :)

Reply:
I think you can just append "Aspect :" to the beginning of the description and fold and print that instead.

Reply:
ok, i cannot seem to be able to do that.
so for now i just put the whole monster description block on a new line.
if someone want to show me how it is possible ( to have that string start on same line as label "aspect : " just after it, and then at newline start but without the offset this time ), i'll be glad for the help and study it.